### PR TITLE
Unlink profiling batch with invoicing batch

### PR DESCRIPTION
**Pull request by sim6**
Adapt documentation to https://github.com/gisce/erp/commit/55cad88f68f24a0ba909d064ef67a35613f0c7c1

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_profiles_unlink_profiling_batch_with_invoicing_batch/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_profiles_unlink_profiling_batch_with_invoicing_batch/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_profiles_unlink_profiling_batch_with_invoicing_batch/

**Comment by sim6** [comment]
Test are falling because missing translations not related with this PR. Do I need to add this translations to request the merge?

**Comment by ecarreras** [comment]
> Test are falling because missing translations not related with this PR. Do I need to add this translations to request the merge?

No, if all of your content is translated I'll merge it

**Comment by sim6** [comment]
@ecarreras ok, thanks

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_profiles_unlink_profiling_batch_with_invoicing_batch/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_profiles_unlink_profiling_batch_with_invoicing_batch/

**Comment by sim6** [comment]
@ecarreras millor amb https://github.com/gisce/powerp-docs/pull/77/commits/34a15e61bf2e0e7deb83a21b7b1c54080cc473a9 ?

**Comment by ecarreras** [comment]
> @ecarreras millor amb 34a15e6 ?

Perfect!

File: docs/distri/perfiles.md [hunk 1]
```diff
@@ -88,48 +88,29 @@ apretant la tecla *ESC*.
 
 La següent comprovació que tenim de fer és:
```
Comment (tinogis) [inline docs/distri/perfiles.md]: ... hem de fer ...

Comment (sim6) [inline docs/distri/perfiles.md]: 9572183c536467ddda9f1139eefa42cc5093fab3

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_profiles_unlink_profiling_batch_with_invoicing_batch/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_profiles_unlink_profiling_batch_with_invoicing_batch/

**Pull request merged**